### DISCUSSION TITLE
Python Notebook Rendering

### DISCUSSION THREAD

**Discussion by RachelBrodsky** [inducer/relate#871] [General]
When trying to render a Python notebook as instructed by the documentation (use {{ render_notebook_cells("code/test.ipynb", clear_markdown=True, clear_output=True) }}) I get an OS input/output error (shown in the image). How do you resolve this? Do you know what this error means?
<img width="1401" alt="Screen Shot 2022-01-03 at 9 58 40 PM" src="https://user-images.githubusercontent.com/67277602/148003815-0aa07b25-469f-486b-9f3f-40da6118ed23.png">

**Comment by dzhuang** [comment]
It seems that means the `code/test.ipynb` not found in the path.

**Reply by inducer** [reply]
Do you mean in the git repo? (That's where I'd expect it to live.)

**Reply by dzhuang** [reply]
> Do you mean in the git repo? (That's where I'd expect it to live.)

Maybe I was wrong (that should have been validated).

**Comment by inducer** [comment]
#872 tracks the addition of a better error message in this case.

@RachelBrodsky could you add the "copy-paste view" of the traceback? That's more helpful for troubleshooting than the (partial) screenshot.

**Reply by dzhuang** [reply]
I think that might be caused by a revision in a fork, since the code line number 284  show different content with what was in the main branch.
P.S., I read [even a `print` might cause that error](https://stackoverflow.com/a/68173741/3437454).
